src/tests/ftest/io/ior_intercept_verify_data_integrity.yaml

```diff
@@ -0,0 +1,54 @@
+hosts:
+    test_servers:
+        - server-A
+        - server-B
+    test_clients:
+        - client-A
+        - client-B
+        - client-C
+        - client-D
+        - client-E
+        - client-F
+timeout: 7200
+server_config:
+    name: daos_server
+    servers:
+        bdev_class: nvme
+        bdev_list: ["0000:5e:00.0","0000:5f:00.0"]
+        scm_size: 35
```

Comment:
This scm_size specifies the size to allocate for the tmpfs when scm is set as 'ram', I don't think this is needed here since you are using dcpm.

Reply:
Ok, will comment it, just in case someone wants to run locally with tmpfs it can be uncommented.

Reply:
Will you be updating this?